tests/dtypes/test_geometry.py

```diff
@@ -150,6 +149,25 @@ def test_geometry_params_srid(
         }
     ]
 
+    resp = app.post(f'/{model}', json={
+        'name': "Vilnius",
+        'coordinates': 'SRID=3346;POINT(582710 6061887)',
+    })
+    assert resp.status_code == 201
+
+    resp = app.get(f'/{model}')
+
+    assert listdata(resp, full=True) == [
+        {
+            'name': "Vilnius",
+            'coordinates': 'POINT (582710 6061887)',
+        },
+        {
+            'name': "Vilnius",
+            'coordinates': 'POINT (582710 6061887)',
+        }
+    ]
```

Comment:
Please, move it to another test, that tests case when SRID is given in geometry itself.